### PR TITLE
Terminus fix

### PR DESCRIPTION
**Pull request by gdraganic**
## Information

| Type                  | Breaking change |
| --------------------- | --------------- |
| Fix | No          |

---
Fix for async terminus handlers.
Make `@godaddy/terminus` a peerDependency.

## Todos

- [ ] Tests
- [ ] Coverage
- [ ] Example
- [ ] Documentation

### CODE REVIEW

File: packages/third-parties/terminus/package.json [hunk 1]
```diff
@@ -31,6 +30,7 @@
     "@tsed/schema": "6.114.0"
   },
   "peerDependencies": {
+    "@godaddy/terminus": "^4.7.1",
```
Comment (Romakita) [inline packages/third-parties/terminus/package.json:33]: Can you revert this change please or add a peerDependenciesMeta pls.
it’ll be a breaking change, if we let that as it ;)

**Review by Romakita — CHANGES_REQUESTED** [review]

File: packages/third-parties/terminus/package.json (hunk 1 above)
Comment (gdraganic) [reply packages/third-parties/terminus/package.json:33]: added :)

**Review by Romakita — APPROVED** [review]
